### ISSUE TITLE
RewardBasedVideoAd.Instance does not work.

### ISSUE THREAD

**Issue by b3da** [googleads/googleads-mobile-unity#785]
I installed the latest version of the plugin (3.15.1) on unity 5.4.1, but no test ads (both interstitial and rewarded video) be shown.
I found somewhere that version 3.4 of the plugin is stable, so I installed it. in this version interstitial banner worked but rewarded no. I put a text on the scene to return the messages. I got "rewarded video failed to load" here.

At last I removed everything and restart unity and again install 3.15.1 version and put some texts on the Start() function. Here is the content of Start() function:

```

void Start () {
	string testApId = "ca-app-pub-3940256099942544~3347511713";
        txtDebug.text = "BeforeAdInit; ";
        // Initialize the Google Mobile Ads SDK.
        MobileAds.Initialize(testApId);

        txtDebug.text += "BeforeGetRewardInstance; ";
	// Get singleton reward based video ad reference.
        this.rewardBasedVideo = RewardBasedVideoAd.Instance;

        txtDebug.text += "BeforeRegisterRewardVideoHandlers; ";
        // RewardBasedVideoAd is a singleton, so handlers should only be registered once.
        this.rewardBasedVideo.OnAdLoaded += this.HandleRewardBasedVideoLoaded;
        this.rewardBasedVideo.OnAdFailedToLoad += this.HandleRewardBasedVideoFailedToLoad;
        this.rewardBasedVideo.OnAdOpening += this.HandleRewardBasedVideoOpened;
        this.rewardBasedVideo.OnAdStarted += this.HandleRewardBasedVideoStarted;
        this.rewardBasedVideo.OnAdRewarded += this.HandleRewardBasedVideoRewarded;
        this.rewardBasedVideo.OnAdClosed += this.HandleRewardBasedVideoClosed;
        this.rewardBasedVideo.OnAdLeavingApplication += this.HandleRewardBasedVideoLeftApplication;

        txtDebug.text += "BeforeRequestInterBanner; ";
	RequestAd(AdUnitIdEnum.InterstitialBanner);

        txtDebug.text += "BeforeRequestRewardedVideo; ";
	RequestAd(AdUnitIdEnum.RewardedVideo);
}
```

The only texts I got from scene was "BeforeAdInit; BeforeGetRewardInstance;".
So, it means than when code reaches to RewardBasedVideoAd.Instance it encounters an error and can't get the instance.

Do you know where the problem is?

Edit: Here's a part of logcat:

```
10-14 16:34:40.590: I/Unity(27536): (Filename: ./artifacts/generated/common/runtime/UnityEngineDebugBindings.gen.cpp Line: 42)
10-14 16:34:40.595: I/Unity(27536): NullReferenceException: Object reference not set to an instance of an object
10-14 16:34:40.595: I/Unity(27536):   at GoogleMobileAds.Api.RewardBasedVideoAd..ctor () [0x00000] in <filename unknown>:0 
10-14 16:34:40.595: I/Unity(27536):   at GoogleMobileAds.Api.RewardBasedVideoAd..cctor () [0x00000] in <filename unknown>:0 
10-14 16:34:40.595: I/Unity(27536): Rethrow as TypeInitializationException: An exception was thrown by the type initializer for GoogleMobileAds.Api.RewardBasedVideoAd
10-14 16:34:40.595: I/Unity(27536):   at AdMobManager.Start () [0x00000] in <filename unknown>:0 
10-14 16:34:40.595: I/Unity(27536): UnityEngine.DebugLogHandler:Internal_LogException(Exception, Object)
10-14 16:34:40.595: I/Unity(27536): UnityEngine.DebugLogHandler:LogException(Exception, Object)
10-14 16:34:40.595: I/Unity(27536): UnityEngine.Logger:LogException(Exception, Object)
10-14 16:34:40.595: I/Unity(27536): UnityEngine.Debug:LogException(Exception)
10-14 16:34:40.595: I/Unity(27536): UnityEngine.EventSystems.ExecuteEvents:Execute(GameObject, BaseEventData, EventFunction`1)
10-14 16:34:40.595: I/Unity(27536): UnityEngine.EventSystems.StandaloneInputModule:ProcessTouchPress(PointerEventData, Boolean, Boolean)
10-14 16:34:40.595: I/Unity(27536): UnityEngine.EventSystems.StandaloneInputModule:ProcessTouchEvents()
10-14 16:34:40.595: I/Unity(27536): UnityEngine.EventSystems.StandaloneInputModule:Process()
10-14 16:34:40.595: I/Unity(27536): UnityEngine.EventSystems.EventSystem:
```

**Comment by rampara** [comment]
@b3da Looks like the rewarded video ad instance is not being instantiated correctly. Might be due to the clashing of files from different versions of the plugin you're tried. Can you remove all plugin related files and then import the latest version?

**Comment by b3da** [comment]
[SOLVED] I used version 3.13.0 of the plugin and it worked.

**Comment by rampara** [comment]
@b3da Did a quick test on my end and is working as expected in a clean project with the latest version fo the plugin.
